public/scripts/acquisition.js

```diff
@@ -27,31 +27,25 @@ function init() {
 
 }
 
+//on keyup, this function is called, but why? what does it do?
 function enterSession(key) {
-		
-
     document.getElementById("sessionInput").style.display = "none";
     document.getElementById("sessionLabel").style.display = "none";
     document.getElementsByClassName("resetButton")[0].style.display = "block";
 
-		
-
-
     let exists = 0;
 		
-	
+	//This call checks to see if the "key" exists inside the database
     let dbexists = app.database().ref().child(key).child("exists");
-	
+
+	//and then updates "exists" as needed
     dbexists.on('value', snap => exists = snap.val());
-		
-	
-	
 
     let uConnected = 0;
     setTimeout(() => {
-
         if (exists != 1) {
             firebase.database().ref().child(key).set({
```

Comment:
If DB entry with the session ID name doesn't exist yet, we populate the DB with initial values